### PR TITLE
Display property images

### PR DESCRIPTION
**Pull request by ricardoruwer**
It just displays the images of the properties on the website pages.

### CODE REVIEW

File: assets/css/page/style.css [hunk 1]
```diff
@@ -162,8 +162,11 @@
 }
 .flat-item-image > a > img {
     width: 100%;
-    -webkit-transition: all 0.8s ease 0s;
-    transition: all 0.8s ease 0s;
+    -webkit-transition: all .8s ease 0s;
```
Comment (rv-andre) [inline assets/css/page/style.css]: vc n precisa passar o valor do `delay` se ele for 0. E qdo for um valor 0, vc tbm nao precisa passar o `s`. <3

File: lib/construtora_lc_hiert/storage/uploaders/image.ex [hunk 2]
```diff
@@ -11,4 +11,24 @@ defmodule ConstrutoraLcHiert.Storage.Uploaders.Image do
...
+
+      iex> full_url(image)
+      "https://ik.imagekit.io/ricardoruwer/lchiert/uploads/1/image.jpg"
+
+  """
+  def full_url(image) do
+    storage_dir = storage_dir(:original, {image.image, image})
+
+    if Mix.env() == :prod do
+      "https://ik.imagekit.io/ricardoruwer/lchiert/#{storage_dir}/#{image.image.file_name}"
```
Comment (andrielfn) [inline lib/construtora_lc_hiert/storage/uploaders/image.ex]: Why your name in the URL?

File: lib/construtora_lc_hiert_web/templates/property/_property_item.html.eex [hunk 3]
```diff
@@ -1,7 +1,9 @@
 <div class="flat-item">
   <div class="flat-item-image">
     <a href="<%= Routes.property_path(@conn, :show, @property.slug) %>">
-      <img src="<%= Routes.static_path(@conn, "/images/flat/1.jpg") %>" alt="">
+      <img
```
Comment (andrielfn) [inline lib/construtora_lc_hiert_web/templates/property/_property_item.html.eex]: `img_tag` instead?

File: lib/construtora_lc_hiert_web/templates/property/show.html.eex [hunk 4]
```diff
@@ -7,60 +7,26 @@
...
-                  </a>
-                </div>
-                <div role="tabpanel" class="tab-pane fade" id="pro-5">
-                  <a href="<%= Routes.static_path(@conn, "/images/flat/1.jpg") %>">
-                    <img src="<%= Routes.static_path(@conn, "/images/flat/1.jpg") %>" alt="">
-                  </a>
-                </div>
+                <%= for image <- @property.images do %>
+                  <div role="tabpanel" class="<%= image_class(image, @property.images) %>" id="<%= image.id %>">
+                    <img
```
Comment (andrielfn) [inline lib/construtora_lc_hiert_web/templates/property/show.html.eex]: `img_tag` instead?

**Pull request merged**